os.path/os/path.py

```diff
@@ -79,6 +79,13 @@ def islink(path):
     except OSError:
         return False
 
+def realpath(path):
+    import ffilib
```

Comment:
Please move the `import` to the top level. The rest of the code can stay in the function for now, I guess.